pkg/rtc/transport.go

```diff
@@ -1944,6 +1944,8 @@ func (t *PCTransport) handleRemoteOfferReceived(sd *webrtc.SessionDescription) e
 }
 
 func (t *PCTransport) handleRemoteAnswerReceived(sd *webrtc.SessionDescription) error {
+	t.clearSignalStateCheckTimer()
+
```

Comment:
Moving this before setting remote description. Reception of remote description is proof that signalling exchange is working. `SetRemoteDescription` error will be handled in event loop failing.